pkgs/os-specific/linux/bitcraze-udev-rules/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "bitcraze-udev-rules";
+  version = "0.1.23";
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/lib/udev/rules.d
+    cp ${./99-bitcraze.rules} $out/lib/udev/rules.d/99-bitcraze.rules
```

[Don't forget the hooks (see checklist) :)](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#new-packages)

Also, creating dirs and moving files can be done in one step with `install`

```suggestion
    runHook preInstall
    
    install -Dm444 ${./99-bitcraze.rules} $out/lib/udev/rules.d/99-bitcraze.rules
    
    runHook postInstall
```